x/auction/keeper/msg_server.go

```diff
@@ -41,9 +41,15 @@ func (k Keeper) SubmitBid(c context.Context, msg *types.MsgSubmitBidRequest) (*t
 	minimumSaleTokenPurchaseAmount := msg.SaleTokenMinimumAmount.Amount
 	maxBidInUsomm := msg.MaxBidInUsomm.Amount
 
+	// To prevent spamming of many small bids, check that minimum bid amount is satisfied (unless amount left in auction is < minimum bid req)
+	minUsommBid := sdk.NewIntFromUint64(k.GetParamSet(ctx).MinimumBidInUsomm)
+
+	if maxBidInUsomm.LT(minUsommBid) && totalSaleTokenBalance.Amount.ToDec().Mul(auction.CurrentUnitPriceInUsomm).GTE(minUsommBid.ToDec()) {
```

Comment:
Couldn't someone still spam the chain with small bids once the total sale token balance value gets below the minimum allowed bid?

Reply:
hm yeah, i guess we can either always require a max bid of the minUsommBid (pretty reasonable I think) and in the case of having less than that just send whats left in the auction, or if the amount remaining is less than minUsommBid we can make the new requirement of the bid to be GTE whats left (also seems fine). Thoughts?

Reply:
Makes sense to me

Reply:
Bumping this to note a fix is still necessary.

Reply:
Updated.